### PR TITLE
Open SSH when workflow failed

### PR DESCRIPTION
**Pull request by KanShiori**
### What is changed and how does it work?

Retain the cluster when workflow failed.

Open SSH for 1 hour when workflow failed.

### Code changes

- [ ] Has Go code change
- [x] Has CI related scripts change

### Tests

- [ ] Unit test 
- [ ] E2E test 
- [ ] Manual test 
- [x] No code 

### Side effects

- [ ] Breaking backward compatibility 
- [ ] Other side effects: 

### Related changes

- [ ] Need to cherry-pick to the release branch 
- [ ] Need to update the documentation 

### Release Notes

Please refer to [Release Notes Language Style Guide](https://github.com/pingcap/tidb-operator/blob/master/docs/release-note-guide.md) before writing the release note.

```release-note

```

### CODE REVIEW

File: tests/examples/001-basic.sh [hunk 1]
```diff
@@ -26,8 +26,6 @@ function cleanup() {
     kubectl delete ns $NS
 }
 
-trap cleanup EXIT
```
Comment (csuzhangxc) [inline tests/examples/001-basic.sh:29]: Do we need to remove `cleanup` too?

Comment (KanShiori) [reply tests/examples/001-basic.sh:29]: ok, it's useless

**Review by DanielZhangQD — APPROVED** [review]
LGTM

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**